src/screens/AccountUtxoBubbles/hooks/useZoomGesture/utils.ts

```diff
@@ -0,0 +1,7 @@
+export const clampScale = (value: number, min: number, max: number) => {
+  return Math.max(min, Math.min(max, value));
+};
+
+export const getScaleFromDimensions = (width: number, height: number) => {
+  return width > height ? (width / height) * 0.8 : (height / width) * 0.8;
```

Comment:
This is a function to get proper scale based on the width and height of the content, but actually we don't need it since we are using the default scale when double tapping. So I have removed it.